### ISSUE TITLE
[url_lower] multi-level routes false positive

### ISSUE THREAD

**Issue by loostro** [QuickenLoans/ramllint#49]
```
error https://acme.com/v1/users/activate/{token}
  RAML section (resource) relativeUri violates: only lowercase letters and underscores allowed [url_lower]
```

This error occurs given such RAML:

```
# main.raml
/users: !include resources/users.raml

# users.raml
/activate/{token}:
    # ... here definitions
```

It disappears when I changed the `users.raml` to:

```
# users.raml
/activate:
  /{token}:
    # ... here definitions
```

This is a false-positive, as `/activate` is not a resource on it's own. It's always `/activate/{token}`.

**Comment by kalisjoshua** [comment]
I might be rusty on my RAML but just because `/activate` is on it's own line doesn't immediately make it a resource in the API; specifically if there are no responses (examples or codes). This would just force separation of URI parts - for good or bad - and might conflict with your desired RAML style.
